Description: Add -mcpu=thunderx2t99 support (documentation)
 * config/aarch64/aarch64-cores.def: Add thunderx2t99.  Change vulcan
 to reference thunderx2t99 for the tuning structure
 * config/aarch64/aarch64-cost-tables.h (vulcan_extra_costs): Rename to ...
 (thunderx2t99_extra_costs): This.
 * config/aarch64/aarch64-tune.md: Regenerate.
 * config/aarch64/aarch64.c (vulcan_addrcost_table): Rename to ...
 (vulcan_addrcost_table): This.
 (vulcan_regmove_cost): Rename to ...
 (thunderx2t99_regmove_cost): This.
 (vulcan_vector_cost): Rename to ...
 (thunderx2t99_vector_cost): this.
 (vulcan_branch_cost): Rename to ...
 (thunderx2t99_branch_cost): This.
 (vulcan_tunings): Rename to ...
 (thunderx2t99_tunings): This and s/vulcan/thunderx2t99 .
 * doc/invoke.texi (AARCH64/mtune): Add thunderx2t99.
 .
 Backported to gcc-5 by dann frazier <dannf@ubuntu.com>
Author: Andrew Pinski <apinski@cavium.com>
Origin: https://gcc.gnu.org/viewcvs/gcc?view=revision&revision=244235
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1727540
Applied-Upstream: svn://gcc.gnu.org/svn/gcc/trunk@244235

diff -urpN a/src/gcc/doc/invoke.texi b/src/gcc/doc/invoke.texi
--- a/src/gcc/doc/invoke.texi	2017-10-20 17:04:39.461376437 -0400
+++ b/src/gcc/doc/invoke.texi	2017-10-20 17:19:39.658847810 -0400
@@ -12412,7 +12412,7 @@ Specify the name of the target processor
 performance of the code.  Permissible values for this option are:
 @samp{generic}, @samp{cortex-a35}, @samp{cortex-a53}, @samp{cortex-a57},
 @samp{cortex-a72}, @samp{exynos-m1}, @samp{qdf24xx}, @samp{thunderx},
-@samp{vulcan}, @samp{xgene1}.
+@samp{thunderx2t99}, @samp{vulcan}, @samp{xgene1}.
 
 Additionally, this option can specify that GCC should tune the performance
 of the code for a big.LITTLE system.  Permissible values for this
